cmd/svcat/binding/bind_cmd.go

```diff
@@ -46,6 +47,7 @@ func NewBindCmd(cxt *command.Context) *cobra.Command {
 		Example: `
   svcat bind wordpress
   svcat bind wordpress-mysql-instance --name wordpress-mysql-binding --secret-name wordpress-mysql-secret
+  svcat bind wordpress-mysql-instance --name wordpress-mysql-binding --external-id wordpress-mysql-binding-external-id
```

Comment:
I think these examples would be more clear if you used a representative value for the external id. Seems like many brokers use UUIDs here, so something like `7e2c42f3-6d94-4409-bb15-7610d60af544` is my preference.